v2/src/components/recipeBoxes/guides.json

```diff
@@ -67,6 +67,11 @@
       "title": "Multi tenancy / organizations",
       "icon": "multitenancy",
       "path": "/docs/multitenancy/introduction"
+    },
+    {
+      "title": "Anomaly Detection",
```

Comment:
Change all the titles to "Bot / Spam detection" since thats what we are going to communicate in other parts of the wbesite.

Reply:
ok cool